bst_plugin/process_nst_OM_from_head.m

```diff
@@ -215,7 +215,7 @@
     segmentation_name = 'segmentation_5tissues';
     iseg = 0;
     for ianat = 1:size(sSubject.Anatomy,2)
-        if any(strcmp(sSubject.Anatomy(ianat).Comment, segmentation_name))
+        if any(contains(sSubject.Anatomy(ianat).Comment, segmentation_name))
```

Comment:
Better to use `~isempty(strfind(...))`
The function `contains` is only available from matlab2016b

Reply:
Thx for the comment.

Reply:
the previous usage of contains allowed 'segmentation_5tissues' to be a **substring** of the anatomy comment.
Now the way you wrote it,  the anatomy comment has to be **exactly** 'segmentation_5tissues'.

Is that what you want?

Reply:
No, it does the same thing. 
`~isempty(strfind('segmentation_5tissues_vox2ras', 'segmentation_5tissues'))` does return 1 as expected